### PR TITLE
[master] Remove <v1.25 pod monitor

### PR DESCRIPTION
**Pull request by rene-dekker**
Avoid keeping lingering object with duplicate monitor name.

### CODE REVIEW

File: pkg/render/monitor/monitor.go
Comment (rene-dekker) [inline pkg/render/monitor/monitor.go:204]: I should have used operator release name...

**Review by tmjd — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Review by tmjd — APPROVED** [review]

**Pull request merged**
